### DISCUSSION TITLE
Announcement: Security Training

### DISCUSSION THREAD

**Discussion by mrybczyn** [eclipse-csi/.github#9] [Announcements]
To support fellow developers, the Eclipse Foundation is offering free security training for all Committers and Contributors.

The complete training contains three parts: the fundamentals, secure coding and advanced topics. We begin with the fundamentals split in two 1h sessions in November 2024. The remaining parts will take place early next year.

What will you learn?

November 20th: 

- How malicious actors attack developers?
- What is the risk-based approach to security?
- How to set up-your workstation?

November 21st:

- How to use multi-factor authentication?
- How to protect your code?
- Bonus: how to report bugs without releasing private information?

Technical topics we’re going to talk about: using hardware tokens, how to create good passwords, setting up your SSH key, protecting your access credentials.

You can register for either or both sessions, each requiring one hour. Upon completing both sessions, you'll earn a special badge! 

Join us for hands-on training, designed to provide practical, real-world security skills, along with the opportunity to engage directly with Eclipse Foundation security experts, discuss a wide array of topics and ask questions in an open, collaborative environment.

This is only the first part of the security training, we’re preparing more advanced topics. Stay tuned!

Register for the session of November 20th: https://eclipse.zoom.us/meeting/register/tZEsf-GrqDwtHNyAJVyiti9oJX1yYWUcdC2I

Register for the session of November 21st: https://eclipse.zoom.us/meeting/register/tZMsfumsrD8rEt3ZLGzwwTfUJNj9r2YS9UkH

**Comment by potiuk** [comment]
Just to clarify - is it for 'Eclipse Foundation' contributors and committers or for "OSS contributors and committers" in-general?

**Reply by mbarbero** [reply]
Anyone can join, but note that we may discuss eclipse foundation specific things though (e.g., policies, projects default setup …)

**Reply by potiuk** [reply]
Thank you

**Comment by IamLRBA** [comment]
Hello @mbarbero , i wish to ask if The Eclipse Foundation security training for all Committers and Contributors is going to be a physical training and if so, will there be a way for other people all around the globe to participate or join online? Because i am currently in Kampala, Uganda and would very much like to be part of it.
Thanks!

**Reply by mbarbero** [reply]
It is an online-only training open to everyone. You are more than welcome to register and join!

**Reply by IamLRBA** [reply]
Thanks, I have just finished registering for both sessions.

**Comment by geziefer** [comment]
@mrybczyn  Hi - how long will each session last?

**Reply by mrybczyn** [reply]
> @mrybczyn Hi - how long will each session last?

We expect less than one hour, including the discussion.

**Comment by kriegaex** [comment]
At my current location, the sessions will start at 10 p.m., which probably clashes with my family time. Is there an option to watch recordings and then maybe do a subsequent quiz to prove that I have actually attended and paid attention? I do not want the badge for free, of course. The content is more important.

**Reply by mrybczyn** [reply]
We will be recording, so that you can watch it later too. We aim at offering the badge later too, the exact format depends on the feedback from this session. This is not a one-time effort, we'll be offering those sessions regularly.

**Reply by kriegaex** [reply]
@mrybczyn, thanks for your feedback. I will be trying to attend the live session, I am just not sure if I can make it. So, it is good to know that there is a fallback, just in case. 🙂

**Reply by mrybczyn** [reply]
@kriegaex you can watch them at https://www.youtube.com/playlist?list=PLy7t4z5SYNaT7JN8FIVTGAy1QUnLTdv9R

**Comment by geziefer** [comment]
@mrybczyn Unfortunately, I was not able to take part. Where will the recording be available?

**Reply by mrybczyn** [reply]
@geziefer We're going to post links here when they're ready (soon)

**Reply by geziefer** [reply]
@mrybczyn Sorry to bother again - but still not uploaded?

**Reply by mrybczyn** [reply]
@geziefer Here they are: https://www.youtube.com/playlist?list=PLy7t4z5SYNaT7JN8FIVTGAy1QUnLTdv9R

**Comment by mrybczyn** [comment]
Thank you everyone who attended our sessions on Nov 20 and 21st. We have a crowd and much engagement. This is great to see that our community cares about security! Stay tuned for resources and invitations to the next parts.

**Reply by IamLRBA** [reply]
> Thank you everyone who attended our sessions on Nov 20 and 21st. We have a crowd and much engagement. This is great to see that our community cares about security! Stay tuned for resources and invitations to the next parts.

@mrybczyn it was a great session. Can't wait to attend another!

**Comment by mrybczyn** [comment]
The videos are available at https://www.youtube.com/playlist?list=PLy7t4z5SYNaT7JN8FIVTGAy1QUnLTdv9R
